app/src/renderer/components/common/NiFieldAddon.vue

```diff
@@ -9,18 +9,18 @@ export default {
 </script>
 
 <style lang="stylus">
-@import '~@/styles/variables.styl'
+@import '~variables'
 
 .ni-field-addon + .ni-field-addon
 .ni-field + .ni-field-addon
 .ni-field-addon + .ni-field
-  margin-left -1px
+  margin-left -1 * px
   position relative
   z-index 1
 
 .ni-field-addon
   background app-bg
-  border 1px solid bc
+  border px solid bc
```

Comment:
0 here?

Reply:
Not zero, but I had `px` a variable for 1 pixel.